### PR TITLE
chore: fix command to work in Powershell for users who don't have WSL installed

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: docs/installation/web-interface.md [hunk 1]
```diff
@@ -12,10 +12,15 @@ nav_order: 2
...
    $ mkdir -p workspace && cd workspace
-   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
+   $ docker run -p 8080:8080 -v "${PWD}:/workspace" -v /var/run/docker.sock:/var/run/docker.sock --rm -it quay.io/konveyor/move2kube-aio:latest
    ```
    Access the UI in `http://localhost:8080/`.
 
+   > Note: There is a known issue when using WSL with the above command.  
+   The CNB containerization option will not be availabe.  
+   Also empty folders may be created in the root directory.  
+   If you are on Windows, consider using Powershell instead of WSL until this is fixed.
```
Comment (ashokponkumar) [inline docs/installation/web-interface.md:22]: Let's remove "Consider using" and replace it with "use"

File: docs/installation/web-interface.md [hunk 2]
```diff
@@ -12,10 +12,15 @@ nav_order: 2
 
    ```console
    $ mkdir -p workspace && cd workspace
-   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
+   $ docker run -p 8080:8080 -v "${PWD}:/workspace" -v /var/run/docker.sock:/var/run/docker.sock --rm -it quay.io/konveyor/move2kube-aio:latest
    ```
    Access the UI in `http://localhost:8080/`.
 
+   > Note: There is a known issue when using WSL with the above command.
```
Comment (ashokponkumar) [inline docs/installation/web-interface.md:19]: "when using the above command in WSL."

File: docs/installation/web-interface.md [hunk 3]
```diff
@@ -12,10 +12,15 @@ nav_order: 2
...
    ```console
    $ mkdir -p workspace && cd workspace
-   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
+   $ docker run -p 8080:8080 -v "${PWD}:/workspace" -v /var/run/docker.sock:/var/run/docker.sock --rm -it quay.io/konveyor/move2kube-aio:latest
    ```
    Access the UI in `http://localhost:8080/`.
 
+   > Note: There is a known issue when using WSL with the above command.  
+   The CNB containerization option will not be availabe.  
+   Also empty folders may be created in the root directory.
```
Comment (ashokponkumar) [inline docs/installation/web-interface.md:21]: Let's be specific and say a "/workspace" empty directory will be created.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: docs/installation/web-interface.md (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply docs/installation/web-interface.md]: done

File: docs/installation/web-interface.md (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply docs/installation/web-interface.md]: done

File: docs/installation/web-interface.md (hunk 3 above)
Comment (HarikrishnanBalagopal) [reply docs/installation/web-interface.md]: done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**